spyder/plugins/editor/widgets/tests/test_warnings.py

```diff
@@ -36,8 +37,20 @@ def test_ignore_warnings(qtbot, lsp_codeeditor):
 
     CONF.set('lsp-server', 'pydocstyle/ignore', 'D100')
     CONF.set('lsp-server', 'pycodestyle/ignore', 'E261')
+
+    # After this call the manager needs to be reinitialized
     manager.update_server_list()
-    qtbot.wait(2000)
+
+    test_ci = os.environ.get('CI_PYTEST', os.environ.get('CI', None))
+    if not test_ci and sys.platform == 'darwin':
```

Comment:
Please change this line to

    if os.environ.get('CI', None) is None and sys.platform == 'darwin':

Reply:
https://github.com/spyder-ide/spyder/blob/master/.travis.yml#L9  Is there a Global environment variable that is defined on all CI tests? I looked at the config on travis yaml and azure.yaml adn got it from there.